requirements.txt

```diff
@@ -0,0 +1,9 @@
+django==3.1.7
+gunicorn==20.0.4
+djangorestframework==3.12.2
+djangorestframework-gis==0.17
+pandas==1.3.4
+pyyaml==5.2
+requests==2.19.1
+google-cloud-storage==1.36.2
+model-mommy~=1.5.1
```

Comment:
model-mommy used only for tests, please create multiple tests files on `./requirements/`.

Reply:
Done f7376ec